src/herder/HerderSCPDriver.cpp

```diff
@@ -322,7 +331,8 @@ HerderSCPDriver::validateValueHelper(uint64_t slotIndex,
 }
 
 SCPDriver::ValidationLevel
-HerderSCPDriver::validateValue(uint64_t slotIndex, Value const& value)
+HerderSCPDriver::validateValue(uint64_t slotIndex, Value const& value,
```

Comment:
let's not make changes to the definition of a "valid" value in SCP (see my comment in the PR for rationale)

Reply:
Reason of changing that was not switch from local time to network time, but my understanding of meaning of  PREFERRED_UPGRADE_DATETIME. I've understand it as: "I will vote for upgrade starting at that time, but if my quorum votes for it earlier, I'll accept that".

I've restored validateValue to previous state, but maybe we should consider changing PREFERRED_UPGRADE_DATETIME name to something more close to it current behavior, like EARLIEST_UPGRADE_DATETIME?